modules/javafx.controls/src/test/java/test/javafx/scene/control/ListCellStartEditTest.java

```diff
@@ -41,36 +41,36 @@
 
 import java.util.Collection;
 import java.util.List;
+import java.util.function.Supplier;
 
 import static java.util.stream.Collectors.toList;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 
 /**
  * Parameterized tests for the {@link ListCell#startEdit()} method of {@link ListCell} and all sub implementations.
- * The {@link CheckBoxListCell} is special as in there the checkbox will be disabled
- * based of the editability.
+ * The {@link CheckBoxListCell} is special as in there the checkbox will be disabled based of the editability.
  */
 @RunWith(Parameterized.class)
 public class ListCellStartEditTest {
 
     private static final boolean[] EDITABLE_STATES = { true, false };
 
     private ListView<String> listView;
-    private final ListCell<String> listCell;
+    private final Supplier<ListCell<String>> listCellSupplier;
 
     @Parameterized.Parameters
     public static Collection<Object[]> data() {
-        return wrapAsObjectArray(List.of(new ListCell<>(), new ComboBoxListCell<>(), new TextFieldListCell<>(),
-                new ChoiceBoxListCell<>(), new CheckBoxListCell<>(obj -> new SimpleBooleanProperty())));
+        return wrapAsObjectArray(List.of(ListCell::new, ComboBoxListCell::new, TextFieldListCell::new,
+                ChoiceBoxListCell::new, () -> new CheckBoxListCell<>(obj -> new SimpleBooleanProperty())));
```

Comment:
the issue is about the specialized cells, so I would suggest to remove the base Cell here, it's fully (maybe :) already

Doing so, makes it also simpler to test the not/existance of the editing ui (which is not yet addressed in the tests)